static/js/notifications.js

```diff
@@ -182,6 +182,10 @@ exports.redraw_title = function () {
     }
 };
 
+exports.reset_realm_default_language = function () {
+    $("#id_realm_default_language").val(page_params.realm_default_language);
+};
+
```

Comment:
I think this probably belongs in `admin.js`, since there's where the field is being displayed.